cmd/openshift/operator/kodata/tekton-addon/addons/01-clustertriggerbindings/github.yaml

```diff
@@ -39,6 +39,8 @@ spec:
     value: $(body.head_commit.message)
   - name: git-repo-url
     value: $(body.repository.url)
+  - name: git-repo-clone-url
+    value: $(body.repository.html_url)
```

Comment:
@savitaashture Is there a is reason to use `html_url`?
IMHO, `clone_url` looks good fit (`"clone_url": "https://github.com/org/repo.git`)

Reply:
no reason but in [github-pullreq](https://github.com/savitaashture/operator/blob/fix_gh_push_ctb_1/cmd/openshift/operator/kodata/tekton-addon/addons/01-clustertriggerbindings/github.yaml#L8) we are using html_url so to keep in sync used that

as both `clone_url` and `html_url` works